x-pack/plugin/esql/compute/gen/src/main/java/org/elasticsearch/compute/gen/AggregatorImplementer.java

```diff
@@ -91,8 +91,7 @@ public AggregatorImplementer(
 
         this.init = requireStaticMethod(
             declarationType,
-            // This should be more restrictive and require org.elasticsearch.compute.aggregation.AggregatorState
-            requirePrimitiveOrImplements(elements, Types.RELEASABLE),
+            requirePrimitiveOrImplements(elements, ClassName.get("org.elasticsearch.compute.aggregation", "AggregatorState")),
```

Comment:
Should we move those to the `Types` constants class? To follow what we do with other types.
Same with the `GroupingAggregatorState` one

Reply:
:+1: